extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -103,6 +107,19 @@ export class AmpSidebar extends AMP.BaseElement {
     this.registerAction('toggle', this.toggle_.bind(this));
     this.registerAction('open', this.open_.bind(this));
     this.registerAction('close', this.close_.bind(this));
+
+    installGlobalClickListenerForDoc(this.getAmpDoc()).addBeforeHandler(e => {
```

Comment:
I don't believe `document-click` is a capture. It'd seem to be illogical to have it as a capture click, since it's a classical bubbling processing.